assets/css/styles.css

```diff
@@ -342,4 +507,4 @@
   .bar-preloader {
     top: 0px;
   }
-}
+}
```

Comment:
Add a new line at the bottom of this file @Neutx

Reply:
check now